site/_docs/history.md

```diff
@@ -56,20 +132,217 @@ other software versions as specified in gradle.properties.
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5408">CALCITE-5408</a>]
+  Return type of `PERCENTILE_CONT` should be the same as sort expression
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5410">CALCITE-5410</a>]
+  Assertion error on `PERCENT_REMAINDER` operator with `DECIMAL` type
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5414">CALCITE-5414</a>]
+  Use `DateTimeUtils` to correctly convert between `java.sql` types and Unix timestamps
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5423">CALCITE-5423</a>]
+  Implement `TIMESTAMP_DIFF` function (compatible with BigQuery)
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5424">CALCITE-5424</a>]
+  Customize handling of literals based on type system
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5430">CALCITE-5430</a>]
+  Implement `IFNULL()` for BigQuery dialect
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5432">CALCITE-5432</a>]
+  Implement BigQuery `TIME_ADD`/`TIME_DIFF`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5436">CALCITE-5436</a>]
+  Implement `DATE_SUB`, `TIME_SUB`, `TIMESTAMP_SUB` (compatible w/ BigQuery)
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5439">CALCITE-5439</a>]
+  Validation of Pivot fails after creating a deep copy of `SqlNode`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5447">CALCITE-5447</a>]
+  Add `DATE_TRUNC` for BigQuery
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5450">CALCITE-5450</a>]
+  Add support for `WEEK(WEEKDAY)` for custom time frames to relevant functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5451">CALCITE-5451</a>]
+  Implement `LPAD()` and `RPAD()` functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5452">CALCITE-5452</a>]
+  Add BigQuery `LENGTH()` as synonym for `CHAR_LENGTH()`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5454">CALCITE-5454</a>]
+  Update BigQuery Conformance for `!=` and `%` operators
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5466">CALCITE-5466</a>]
+  Constant condition can't be reduced after correlate
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5471">CALCITE-5471</a>]
+  `RelSupplier.SqlRelSupplier#apply` should use `.project()`, not `.rel`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5489">CALCITE-5489</a>]
+  When creating a `RexCall` to `TIMESTAMP_DIFF` function, cannot convert a `TIMESTAMP` literal to a `org.apache.calcite.avatica.util.TimeUnit`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5491">CALCITE-5491</a>]
+  Allow `TIME` and `DATE` to be args for `TIMESTAMPDIFF`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5493">CALCITE-5493</a>]
+  Time zone tests in `SqlFunctions` should pass in `Europe/London`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5495">CALCITE-5495</a>]
+  Allow `WEEK` and `QUARTER` in `INTERVAL` literals
```

Comment:
5495 is a feature

5151 LPAD, RPAD is a feature (don't lump them into BigQuery, because they are also for Oracle)

5447 DATE_TRUNC, 5432 TIME_ADD/TIME_DIFF, 5430 IFNULL and many other BQ functions are features. There are so many BQ functions that I would have an indented list in the features section

Reply:
I'll make that change. Good idea on the BigQuery section